common/common-rest/src/main/java/org/apache/servicecomb/common/rest/codec/RestObjectMapper.java

```diff
@@ -17,25 +17,37 @@
 
 package org.apache.servicecomb.common.rest.codec;
 
+import java.io.IOException;
 import java.text.FieldPosition;
 import java.util.Date;
 
+import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.core.JsonParser.Feature;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
 import com.fasterxml.jackson.databind.type.TypeFactory;
 
-public final class RestObjectMapper extends ObjectMapper {
-  public static final RestObjectMapper INSTANCE = new RestObjectMapper();
+import io.vertx.core.json.JsonObject;
+
+public class RestObjectMapper extends AbstractRestObjectMapper {
+  private static class JsonObjectSerializer extends JsonSerializer<JsonObject> {
```

Comment:
seems someone already use field "INSTANCE"
so just add a setter method?

Reply:
This code looks bad. And I think if others use this, they can change it easily as this PR has done. So it's not a problem.